validate/validate.go

```diff
@@ -423,25 +396,30 @@ func (v *Validator) CheckMounts() (msgs []string) {
 	return
 }
 
-// CheckOS checks v.spec.Platform.OS
-func (v *Validator) CheckOS() (msgs []string) {
-	logrus.Debugf("check os")
+// CheckPlatform checks v.platform
+func (v *Validator) CheckPlatform() (msgs []string) {
+	logrus.Debugf("check platform")
+
+	if v.platform != "linux" && v.platform != "solaris" && v.platform != "windows" {
```

Comment:
So, spec just supports linux, solaris and windows, all other platforms are invalid?